src/components/SettingDialog.vue

```diff
@@ -714,6 +724,15 @@ export default defineComponent({
       set: (val) => emit("update:modelValue", val),
     });
 
+    const allEngineCanUseGPU = computed({
+      get: () => store.state.allEngineCanUseGPU,
+      set: () => {
+        undefined;
+      },
+    });
```

Comment:
```suggestion
    const allEngineCanUseGPU = computed(() => store.state.allEngineCanUseGPU);
```
setが不要になったので、こう書けば綺麗になると思います...!